docs/serverless/08-02-create-git-function.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Create a Function from Git Repository
+type: Tutorials
+---
+
+This tutorial shows how you can create a simple "Hello World!" Function from Git repository.
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="cli">
+  CLI
+  </summary>
+
+1. Export these variables:
+
+    ```bash
+    export NAME={FUNCTION_NAME}
+    export NAMESPACE={FUNCTION_NAMESPACE}
+    ```
+
+2. Create a GitRepository CR that specifies the Git repository metadata:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha1
+    kind: GitRepository
+    metadata:
+      name: $NAME
+    spec:
+      url: "https://github.com/pPrecel/public-gitops"
```

Comment:
Please use the https://github.com/kyma-project/examples.git repository. It's better to ref to repo in organisation, not to repo in account one of our contributors.